### PR TITLE
Update New-CsTeamsEventsPolicy.md

### PR DESCRIPTION
**Pull request by gregorionMS**
Updated EventAccessType to correctly reflect admin settings for Webinar and Town hall

### CODE REVIEW

**Comment by gregorionMS** [comment]
Added clarity around EventAccessType to account for impact for Town halls for In Organization and Public 

_This setting governs which users can access the event for Town hall and access the event registration page or the event site to register for Webinar. It also governs which user type is allowed to join the session/s in the event for both event types. Possible values are:_

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Review by get-itips — COMMENTED** [review]
Hello @gregorionMS how are you?
Are you a Microsoft employee? If so, can you please link your GH profile to your Microsoft profile so a blue Microsoft submitter badge is added?
Thanks

**Comment by gregorionMS** [comment]
I am a Microsoft employee, I will attempt to do that right now thank you

Jigesh Parekh

From: Andres Mariano Gorzelany ***@***.***>
Date: Tuesday, October 10, 2023 at 10:55 AM
To: MicrosoftDocs/office-docs-powershell ***@***.***>
Cc: Jigesh Parekh ***@***.***>, Mention ***@***.***>
Subject: Re: [MicrosoftDocs/office-docs-powershell] Update New-CsTeamsEventsPolicy.md (PR #11358)

@get-itips commented on this pull request.

Hello @gregorionMS<https://github.com/gregorionMS> how are you?
Are you a Microsoft employee? If so, can you please link your GH profile to your Microsoft profile so a blue Microsoft submitter badge is added?
Thanks

**Comment by get-itips** [comment]
@gregorionMS Thanks, and please make the same edit for Set-CsTeamsEventsPolicy

**Comment by gregorionMS** [comment]
I will, Im trying to find some instructions online on how to link the profiles but cant seem to get any clarity, can you provide some instructions on how to link these profiles?

Thank you for your help on this

Jigesh Parekh

From: Andres Mariano Gorzelany ***@***.***>
Date: Tuesday, October 10, 2023 at 11:09 AM
To: MicrosoftDocs/office-docs-powershell ***@***.***>
Cc: Jigesh Parekh ***@***.***>, Mention ***@***.***>
Subject: Re: [MicrosoftDocs/office-docs-powershell] Update New-CsTeamsEventsPolicy.md (PR #11358)

@gregorionMS<https://github.com/gregorionMS> Thanks, and please make the same edit for Set-CsTeamsEventsPolicy

**Comment by get-itips** [comment]
I think this is the link https://review.docs.microsoft.com/en-us/office-authoring-guide/contribute-get-started-setup-github-ro?branch=main

**Comment by gregorionMS** [comment]
Thanks man, I think im good to go.

I submitted another PR

Update New-CsTeamsEventsPolicy.md by gregorionMS · Pull Request #11360 · MicrosoftDocs/office-docs-powershell (github.com)<https://github.com/MicrosoftDocs/office-docs-powershell/pull/11360>


Thank you again

Jigesh

From: Andres Mariano Gorzelany ***@***.***>
Date: Tuesday, October 10, 2023 at 11:13 AM
To: MicrosoftDocs/office-docs-powershell ***@***.***>
Cc: Jigesh Parekh ***@***.***>, Mention ***@***.***>
Subject: Re: [MicrosoftDocs/office-docs-powershell] Update New-CsTeamsEventsPolicy.md (PR #11358)

I think this is the link https://review.docs.microsoft.com/en-us/office-authoring-guide/contribute-get-started-setup-github-ro?branch=main

**Comment by get-itips** [comment]
@gregorionMS The new PR also edits the same article; for the New-

**Comment by gregorionMS** [comment]
Apologies, good catch- just pushed the changes

[A screenshot of a computer  Description automatically generated]
From: Andres Mariano Gorzelany ***@***.***>
Date: Tuesday, October 10, 2023 at 12:15 PM
To: MicrosoftDocs/office-docs-powershell ***@***.***>
Cc: Jigesh Parekh ***@***.***>, Mention ***@***.***>
Subject: Re: [MicrosoftDocs/office-docs-powershell] Update New-CsTeamsEventsPolicy.md (PR #11358)

@gregorionMS<https://github.com/gregorionMS> The new PR also edits the same article; for the New-

**Comment by get-itips** [comment]
Can you add the changes to this PR that edits those 2 articles? If you do not know how to, no problem.

File: teams/teams-ps/teams/New-CsTeamsEventsPolicy.md [hunk 1]
```diff
@@ -100,11 +100,11 @@ Accept wildcard characters: False
 ```
 
 ### -EventAccessType
-This setting governs which users can access the event registration page or the event site to register. It also governs which user type is allowed to join the session/s in the event. 
+This setting governs which users can access the event for Town hall and access the event registration page or the event site to register for Webinar. It also governs which user type is allowed to join the session/s in the event for both event types.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/New-CsTeamsEventsPolicy.md:103]:
```suggestion
This setting governs which users can access the Town hall event and access the event registration page or the event site to register for a Webinar. It also governs which user type is allowed to join the session or sessions in the event for both event types. 

```

File: teams/teams-ps/teams/New-CsTeamsEventsPolicy.md [hunk 2]
```diff
@@ -100,11 +100,11 @@ Accept wildcard characters: False
 ```
 
 ### -EventAccessType
-This setting governs which users can access the event registration page or the event site to register. It also governs which user type is allowed to join the session/s in the event. 
+This setting governs which users can access the event for Town hall and access the event registration page or the event site to register for Webinar. It also governs which user type is allowed to join the session/s in the event for both event types. 
 Possible values are:
  - **Everyone**: Enables creating events to allow in-tenant, guests, federated, and anonymous (external to the tenant) users to register and join the event.
 
- - **EveryoneInCompanyExcludingGuests**: Enables creating events to allow only in-tenant users to register and join the event.
+ - **EveryoneInCompanyExcludingGuests**: For Webinar - enables creating events to allow only in-tenant users to register and join the event. For Town hall - enables creating events to allow only in-tenant users to join the event (Note: for Town hall, in-tenant users includes guests; this parameter will disable public Town halls).
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/New-CsTeamsEventsPolicy.md:107]:
```suggestion
 - **EveryoneInCompanyExcludingGuests**: For Webinar - enables creating events to allow only in-tenant users to register and join the event. For Town hall - enables creating events to allow only in-tenant users to join the event (Note: for Town hall, in-tenant users include guests; this parameter will disable public Town halls). 

```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @gregorionMS kindly commit the minor suggested changes. Many thanks.

**Review by get-itips — APPROVED** [review]
I'm ok after suggestions are applied

**Comment by gregorionMS** [comment]
Thank you all for the help, when can we see the proposed changes pushed live?

All the best

Jigesh Parekh

From: Andres Mariano Gorzelany ***@***.***>
Date: Wednesday, October 11, 2023 at 8:43 AM
To: MicrosoftDocs/office-docs-powershell ***@***.***>
Cc: Jigesh Parekh ***@***.***>, Mention ***@***.***>
Subject: Re: [MicrosoftDocs/office-docs-powershell] Update New-CsTeamsEventsPolicy.md (PR #11358)

@get-itips approved this pull request.

I'm ok after suggestions are applied

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**